include/bluetooth/hci_vs.h

```diff
@@ -37,6 +48,120 @@ struct bt_hci_rp_vs_read_supported_features {
 	u8_t  features[8];
 } __packed;
 
+#define BT_HCI_OP_VS_SET_EVENT_MASK             BT_OP(BT_OGF_VS, 0x0004)
+struct bt_hci_cp_vs_set_event_mask {
+	u8_t  event_mask[8];
+} __packed;
+
+#define BT_HCI_VS_RESET_SOFT                    0x00
+#define BT_HCI_VS_RESET_HARD                    0x01
+#define BT_HCI_OP_VS_RESET                      BT_OP(BT_OGF_VS, 0x0005)
+struct bt_hci_cp_vs_reset {
+	u8_t  type;
+} __packed;
+
+#define BT_HCI_OP_VS_WRITE_BD_ADDR              BT_OP(BT_OGF_VS, 0x0006)
+struct bt_hci_cp_vs_write_bd_addr {
+	bt_addr_t bdaddr;
+} __packed;
+
+#define BT_HCI_VS_TRACE_DISABLED                0x00
+#define BT_HCI_VS_TRACE_ENABLED                 0x01
+
+#define BT_HCI_VS_TRACE_HCI_EVTS                0x00
+#define BT_HCI_VS_TRACE_VDC                     0x01
+#define BT_HCI_OP_VS_SET_TRACE_ENABLE           BT_OP(BT_OGF_VS, 0x0007)
+struct bt_hci_cp_vs_set_trace_enable {
+	u8_t  enable;
+	u8_t  type;
+} __packed;
+
+#define BT_HCI_OP_VS_READ_BUILD_INFO            BT_OP(BT_OGF_VS, 0x0008)
+struct bt_hci_rp_vs_read_build_info {
+	u8_t  status;
+	u8_t  info[0];
+} __packed;
+
+struct bt_hci_vs_static_addr {
+	bt_addr_t bdaddr;
+	u8_t      irk[16];
+} __packed;
+
+#define BT_HCI_OP_VS_READ_STATIC_ADDRS          BT_OP(BT_OGF_VS, 0x0009)
+struct bt_hci_rp_vs_read_static_addrs {
+	u8_t   status;
+	u8_t   num_addrs;
+	struct bt_hci_vs_static_addr a[0];
+} __packed;
+
+#define BT_HCI_OP_VS_READ_KEY_HIERARCHY_ROOTS   BT_OP(BT_OGF_VS, 0x000a)
+struct bt_hci_rp_vs_read_key_hierarchy_roots {
+	u8_t  status;
+	u8_t  irk[16];
+	u8_t  er[16];
+} __packed;
+
+#define BT_HCI_OP_VS_READ_CHIP_TEMP             BT_OP(BT_OGF_VS, 0x000b)
+struct bt_hci_rp_vs_read_chip_temp {
+	u8_t  status;
+	s8_t  temps;
+} __packed;
+
+struct bt_hci_vs_cmd {
+	u16_t vendor_id;
+	u16_t opcode_base;
+} __packed;
+
+#define BT_HCI_VS_VID_ANDROID                   0x0001
+#define BT_HCI_VS_VID_MICROSOFT                 0x0002
+#define BT_HCI_OP_VS_READ_HOST_STACK_CMDS       BT_OP(BT_OGF_VS, 0x000c)
+struct bt_hci_rp_vs_read_host_stack_cmds {
+	u8_t   status;
+	u8_t   num_cmds;
+	struct bt_hci_vs_cmd c[0];
+} __packed;
+
+#define BT_HCI_VS_SCAN_REQ_REPORTS_DISABLED     0x00
+#define BT_HCI_VS_SCAN_REQ_REPORTS_ENABLED      0x01
+#define BT_HCI_OP_VS_SET_SCAN_REQ_REPORTS       BT_OP(BT_OGF_VS, 0x000d)
+struct bt_hci_cp_vs_set_scan_req_reports {
+	u8_t  enable;
+} __packed;
+
+/* Events */
+
+struct bt_hci_evt_vs {
+	u8_t  subevent;
```

Comment:
@holtmann I know you said to keep this in each event, but we don't do that for the LE meta event and so for consistency I prefer to leave it like this.